.github/workflows/integration_tests.yml

```diff
@@ -118,4 +116,139 @@ jobs:
           timeout_minutes: 10
           max_attempts: 3
           command: |
-            python scripts/gha/unity_installer.py --release_license --logfile testapps/release_license.log --version ${{ matrix.unity_version }}
+            python scripts/gha/unity_installer.py --release_license \
+              --logfile testapps/release_license.log \
+              --version ${{ matrix.unity_version }}
+      - name: Upload Android integration tests artifact
```

Comment:
Anyone knows how to better handle the artifacts for different platforms. 
(Have a script is an option.)

Reply:
Would this [composite action](https://github.com/actions/runner/issues/646#issuecomment-901336347) work for you?

Essentially, we can create our own template action that executes other actions such as checking out code, setting up environment, running test, and uploading artifacts, where the step of running test can be customized. Then when we use this template action in the workflow yaml file, we only need to specify the running test part, while artifact uploading is configured implicitly.

Reply:
I assume you mean how to reduce having to duplicate it a bunch?  One potential way is to make all of the platform settings variables that get set as part of the matrix, though that can also be confusing to read.  I think starting with a readable, if a bit wordy, solution is fine for now.